src/converter.ts

```diff
@@ -56,37 +52,71 @@ export type LwM2MAssetTrackerV2 = {
 }
 
 /**
- * Transform the device twin object coming from Azure to an object with LwM2M objects that are required by Asset Tracker v2
+ * The id of the Asset Tracker v2 objects given by Coiote
+ */
+const coioteIds = {
+	Device: 3,
+	ConnectivityMonitoring: 4,
+	Location: 6,
+	Temperature: 3303,
+	Humidity: 3304,
+	Pressure: 3323,
+	Config: 50009,
+}
+
+/**
+ * Convert 'Coiote Asset Tracker v2' format into 'LwM2M Asset Tracker v2' format
  */
 export const converter = async (
 	deviceTwin: DeviceTwin,
-	onWarning?: (element: Warning) => void,
+	onWarning?: (element: UndefinedCoioteObjectWarning) => void,
 	onError?: (element: LwM2MFormatError) => void,
 ): Promise<LwM2MAssetTrackerV2> => {
+	const conversionResult = {} as any //as LwM2MAssetTrackerV2 . TODO: solve this
```

Comment:
**Suggestion:** Avoid using 'any' type. It defeats the purpose of using TypeScript, which is to have type safety. Use a more specific type or create a new type if necessary.
```suggestion
 const conversionResult: Partial<LwM2MAssetTrackerV2> = {}
```

Reply:
This is going to be solved by removing the generic method `convertToLwM2M` and creating a custom method for each case. Until then, this going to be this way.